### DISCUSSION TITLE
Broken link to Geist font license

### DISCUSSION THREAD

**Discussion by dysphasia** [vercel/community#5692] [Bug Report]
### Page to Investigate

https://vercel.com/font/mono

### Steps to Reproduce

The "Get the .zip" section of https://vercel.com/font/mono contains a link to the font license which has an incorrect URL, due to the extension being capitalized.

Current Link: https://github.com/vercel/geist-font/blob/main/LICENSE.TXT

Correct Link: https://github.com/vercel/geist-font/blob/main/LICENSE.txt

**Comment by swarnava** [comment]
Hi,

Thank you for taking time to report this. We have forwarded this internally to right team.

**Comment by swarnava** [comment]
This should be fixed now!
